anyway/widgets/suburban_widgets/accident_count_by_accident_year_widget.py

```diff
@@ -2,17 +2,17 @@
 
 from flask_babel import _
 
-from anyway.request_params import RequestParams
 from anyway.backend_constants import AccidentSeverity
 from anyway.infographics_dictionaries import segment_dictionary
 from anyway.models import AccidentMarkerView
-from anyway.widgets.widget import register
+from anyway.request_params import RequestParams
 from anyway.widgets.suburban_widgets.sub_urban_widget import SubUrbanWidget
+from anyway.widgets.widget import register
 from anyway.widgets.widget_utils import (
     get_accidents_stats,
-    add_empty_keys_to_gen_two_level_dict,
```

Comment:
can this be deleted now?

Reply:
I see that this function is still being used in other places, like `killed_and_injured_count_per_age_group_stacked`